### ISSUE TITLE
Init Runtime: Runtime in timer.dm,437: list index out of bounds

### ISSUE THREAD

**Issue by Neerti** [PolarisSS13/Polaris#6815]
#### Brief description of the issue
During a Travis build in my PR to fail PRs that make runtimes happen, this runtime happened, and like a good Travis, failed the PR making it fail PRs that runtime.

#### What you expected to happen
No runtimes to happen during initialization.

#### What actually happened
```
[2020-03-11T05:28:12] Runtime in timer.dm,437: list index out of bounds
  proc name: bucketJoin (/datum/timedevent/proc/bucketJoin)
  src: Timer: -1 (\[0x21038998]), TTR... (/datum/timedevent)
  call stack:
  Timer: -1 (\[0x21038998]), TTR... (/datum/timedevent): bucketJoin()
  Timer: -1 (\[0x21038998]), TTR... (/datum/timedevent): New(/datum/callback (/datum/callback), 10.2, 0, null)
  addtimer(/datum/callback (/datum/callback), 10.2, 0)
  /datum/looping_sound/air_pump (/datum/looping_sound/air_pump): on start()
  /datum/looping_sound/air_pump (/datum/looping_sound/air_pump): start(null)
  Xenoarcheology Expedition Prep... (/obj/machinery/atmospherics/unary/vent_pump/on): can pump()
  Xenoarcheology Expedition Prep... (/obj/machinery/atmospherics/unary/vent_pump/on): process(20)
  Machines (/datum/controller/subsystem/machines): process machinery(1)
  Machines (/datum/controller/subsystem/machines): fire(1)
  Machines (/datum/controller/subsystem/machines): ignite(1)
  Master (/datum/controller/master): RunQueue()
  Master (/datum/controller/master): Loop()
  Master (/datum/controller/master): StartProcessing(0)
```
A lot of other vents also did this, but the runtimes were almost the same, only difference was a different vent making the runtime. It said 231 runtimes happened.

#### Steps to reproduce
Uncertain.

This might be a runtime that happens intermittently. Figured I'd post it here and restart the job to see if it goes away. If it does that's a bigger problem.

#### Additional info:
 - **Server Revision**: Found using the "Show Server Revision" verb under the OOC tab.
Running revision:
ffe896b3c1df94e7f249145f

6b1589490ea93ae6ffe896b3c1df94e7f249145f
 - **Anything else you may wish to add** (Location if it's a mapping issue, etc)

**Comment by Atermonera** [comment]
Possibly fixed by #6805 ?

**Comment by Neerti** [comment]
The runtime shows up in Travis for PR #6871 , so I think it's still out there.
